src/Element/CiviremoteFundingDashboardGroup.php

```diff
@@ -36,6 +36,8 @@ public function getInfo(): array {
       '#attributes' => [
         'class' => [],
       ],
+      '#prefix' => NULL,
+      '#suffix' => NULL,
```

Comment:
Why is initializing them with `NULL` necessary?

Reply:
This avoids an extra existence check in `preRenderDashboardGroup()`.